### PR TITLE
[Fleet] Update template and packaging code for fleet packages

### PR DESCRIPTION
**Pull request by rw-access**
## Issues
Partial dependency of https://github.com/elastic/detection-rules/issues/1206

## Summary
Noticed some drift since https://github.com/elastic/integrations/tree/master/packages/security_detection_engine. Also since 7.13 has been released, we need to get a 0.13.0 package out to replace the temporary 0.0.2 package.

### CODE REVIEW

File: etc/packages.yml [hunk 1]
```diff
@@ -24,9 +24,20 @@ package:
...
       kibana_version: "^7.13.0"
-    # this determines the version for the package-storage generated artifact
-    version: "0.0.1-dev.3"
+    description: "Prebuilt detection rules for Elastic Security"
+    format_version: "1.0.0"
+    icons:
+    - src: "/img/security-logo-color-64px.svg"
+      size: "16x16"
+      type: "image/svg+xml"
+    license: basic
```
Comment (brokensound77) [inline etc/packages.yml:36]: internal is currently being set to `null`

```suggestion
    internal: true
    license: basic
```

File: etc/security-logo-color-64px.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (brokensound77) [inline etc/security-logo-color-64px.svg:1]: did you test the rendering of this image?

File: etc/packages.yml (hunk 1 above)
Comment (rw-access) [reply etc/packages.yml:36]: that's a good thing. it's not supposed to be internal anymore

File: etc/security-logo-color-64px.svg (hunk 2 above)
Comment (rw-access) [reply etc/security-logo-color-64px.svg:1]: copied it straight out of the endpoint-package repo

**Review by brokensound77 — APPROVED** [review]

**Review by threat-punter — APPROVED** [review]

**Pull request merged**